e2e/setup/global-setup.ts

```
@@ -0,0 +1,238 @@
+    )
+  )
+}
+
+export default async function globalSetup(): Promise<void> {
+  console.log('[e2e] Cleaning up previous containers...')
+  execSync(`docker compose -f ${COMPOSE_FILE} down -v`, {
+    encoding: 'utf-8',
+    cwd: process.cwd()
+  })
+
+  console.log('[e2e] Starting Docker containers...')
```

ESLint

[error] Unexpected console statement.

(no-console)